object-browser-schema.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  additionalProperties: false,
+  definitions: {
+    actionItemDisabled: {
+      description: 'Tells if action bar button or link is disabled or ties it to selection',
+      oneOf: [
+        {
+          type: 'boolean',
+          default: false,
+        }, {
+          type: 'object',
+          additionalProperties: false,
+          description: 'Shortcut to enable and disable button or link as selection changes',
+          properties: {
+            minSelected: {type: 'number'},
+            maxSelected: {type: 'number'}
+          }
+        }
+      ]
+    },
+    actionItem: {
+      type: 'object',
+      properties: {
+        id: {
```

Comment:
We should talk more about what actions get triggered. My assumption was that all buttons and links trigger the same action. That action gets the id as an argument and can decide what to do based on that.